mmf/configs/defaults.yaml

```diff
@@ -135,6 +133,10 @@ training:
 evaluation:
     # Metrics for evaluation
     metrics: []
+    # Generate predictions in a file
+    predict: false
```

Comment:
Shouldn't this by default be true in case of of predict command. It feels weird to ask user to pass this when they are explicitly saying predict.

Reply:
I will set this by default when people run `mmf_predict` cmd.